### PR TITLE
Create OnNPCDisconnect.md

### PR DESCRIPTION
**Pull request by Fede5432**
ES language

### CODE REVIEW

**Review by adib-yg — CHANGES_REQUESTED** [review]
`docs/translations/es/scripting/callbacks/OnNPCDisconnect.md/OnNPCDisconnect.md`

Please correct the file path.

**Comment by Fede5432** [comment]
done

File: docs/translations/es/scripting/callbacks/OnNPCDisconnect.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+título: OnNPCDisconnect
```
Comment (adib-yg) [inline docs/translations/es/scripting/callbacks/OnNPCDisconnect.md:2]:
```suggestion
title: OnNPCDisconnect
```

File: docs/translations/es/scripting/callbacks/OnNPCDisconnect.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+descripción: Este callback se llama cuando un NPC se desconecta del servidor.
```
Comment (adib-yg) [inline docs/translations/es/scripting/callbacks/OnNPCDisconnect.md:3]:
```suggestion
description: Este callback se llama cuando un NPC se desconecta del servidor.
```

**Review by adib-yg — CHANGES_REQUESTED** [review]

**Review by adib-yg — APPROVED** [review]

**Pull request merged**